### ISSUE TITLE
no 'string' command in ubuntu

### ISSUE THREAD

**Issue by seagle0128** [fisherman/g2#2]
OS: Ubuntu 16.04 LTS
G2 is based on git version 2.7.4 and fish version 2.2.0
 
``` shell
~/.emacs.d: g sync
No command 'string' found, did you mean:
 Command 'spring' from package 'ruby-spring' (universe)
 Command 'spring' from package 'spring' (universe)
 Command 'strings' from package 'binutils-multiarch' (main)
 Command 'strings' from package 'binutils' (main)
string: command not found
~/.config/fish/conf.d/g2.fish (line 1): if command git log --oneline -1 --pretty=format:'%s' ^/dev/null | string match -q -i WIPWIPWIPWIP
```

**Comment by jbucaran** [comment]
`g2` is using the string command which is only available in fish >=2.3.

You are just one minor behind ;)

**Comment by seagle0128** [comment]
thx, @jbucaran .
